selfdrive/car/honda/interface.py

```diff
@@ -161,10 +161,12 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.lateralParams.torqueBP, ret.lateralParams.torqueV = [[0, 3840], [0, 3840]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.2], [0.06]]
 
-    elif candidate in (CAR.HONDA_ODYSSEY, CAR.HONDA_ODYSSEY_CHN):
+    elif candidate in (CAR.HONDA_ODYSSEY, CAR.HONDA_ODYSSEY_CHN, CAR.HONDA_ODYSSEY_BOSCH):
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.28], [0.08]]
       if candidate == CAR.HONDA_ODYSSEY_CHN:
         ret.lateralParams.torqueBP, ret.lateralParams.torqueV = [[0, 32767], [0, 32767]]  # TODO: determine if there is a dead zone at the top end
+      elif not ret.openpilotLongitudinalControl:
+        ret.lateralParams.torqueBP, ret.lateralParams.torqueV = [[0, 3840], [0, 3840]]  # clipped by radar
```

Comment:
This may be a little too clever. I had to take time to read other Honda car port code to be sure this couldn't affect HONDA_ODYSSEY, and it might not be robust in the face of new Honda car ports being added. It would be clearer to make this conditional on HONDA_ODYSSEY_BOSCH and keep the comment.

```suggestion
      elif candidate == CAR.HONDA_ODYSSEY_BOSCH:
        ret.lateralParams.torqueBP, ret.lateralParams.torqueV = [[0, 3840], [0, 3840]]  # clipped by radar
```